BBallStatsTrack/src/com/example/bballstatstrack/models/Game.java

```diff
@@ -0,0 +1,406 @@
+package com.example.bballstatstrack.models;
+
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import com.example.bballstatstrack.models.game.GameLog;
+import com.example.bballstatstrack.models.gameevents.GameEvent;
+import com.example.bballstatstrack.models.gameevents.TurnoverEvent;
+import com.example.bballstatstrack.models.gameevents.foulevents.NonShootingFoulEvent;
+import com.example.bballstatstrack.models.gameevents.foulevents.ShootingFoulEvent;
+
+import android.text.format.DateFormat;
+
+public class Game
```

Comment:
Game can actually be implemented as a Singleton since there will always be a single instance of this on the app. Making this a singleton will let you lessen the times Game is passed in the parameter.